### PR TITLE
Create README

### PR DESCRIPTION
**Pull request by meaghanlewis**
Because projects _need_ readme's

### CODE REVIEW

File: TestWebAppProject/Startup.cs [hunk 1]
```diff
@@ -18,6 +18,9 @@ public Startup(IHostingEnvironment env)
                 .SetBasePath(env.ContentRootPath)
                 .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
                 .AddJsonFile($"appsettings.{env.EnvironmentName}.json", optional: true)
+
+
+                //This is a comment
```
Comment (ungb) [inline TestWebAppProject/Startup.cs]: What do you guys think about this change? https://github.com/github/VisualStudio/pull/1150

Comment (meaghanlewis) [reply TestWebAppProject/Startup.cs]: LGTM

File: TestWebAppProject/Startup.cs [hunk 2]
```diff
@@ -18,6 +18,9 @@ public Startup(IHostingEnvironment env)
                 .SetBasePath(env.ContentRootPath)
                 .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
                 .AddJsonFile($"appsettings.{env.EnvironmentName}.json", optional: true)
+
```
Comment (meaghanlewis) [inline TestWebAppProject/Startup.cs]: Remove this line

Comment (meaghanlewis) [inline TestWebAppProject/Startup.cs]: Remove this line

Comment (meaghanlewis) [inline TestWebAppProject/Startup.cs]: bbbb

File: TestWebAppProject/Startup.cs [hunk 3]
```diff
@@ -18,6 +18,9 @@ public Startup(IHostingEnvironment env)
...
+
```
Comment (ungb) [inline TestWebAppProject/Startup.cs]: fsdfew

File: TestWebAppProject/Startup.cs (hunk 2 above)
Comment (meaghanlewis) [inline TestWebAppProject/Startup.cs]: blah

File: TestWebAppProject/Startup.cs (hunk 3 above)
Comment (meaghanlewis) [reply TestWebAppProject/Startup.cs]: replying

File: TestWebAppProject/Startup.cs [hunk 4]
```diff
@@ -18,6 +18,9 @@ public Startup(IHostingEnvironment env)
                 .SetBasePath(env.ContentRootPath)
                 .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
                 .AddJsonFile($"appsettings.{env.EnvironmentName}.json", optional: true)
```
Comment (meaghanlewis) [inline TestWebAppProject/Startup.cs]: bgtygv

File: TestWebAppProject/Startup.cs (hunk 2 above)
Comment (meaghanlewis) [inline TestWebAppProject/Startup.cs]: vhgvgh

Comment (ungb) [inline TestWebAppProject/Startup.cs]: werewr

Comment (meaghanlewis) [inline TestWebAppProject/Startup.cs]: boo

File: TestWebAppProject/Startup.cs [hunk 5]
```diff
@@ -31,6 +34,15 @@ public void ConfigureServices(IServiceCollection services)
             services.AddMvc();
         }
 
+
```
Comment (ungb) [inline TestWebAppProject/Startup.cs]: test

File: TestWebAppProject/Startup.cs (hunk 2 above)
Comment (ungb) [inline TestWebAppProject/Startup.cs]: etst

Comment (ungb) [inline TestWebAppProject/Startup.cs]: testing this

File: TestWebAppProject/Startup.cs [hunk 6]
```diff
@@ -31,6 +34,15 @@ public void ConfigureServices(IServiceCollection services)
...
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="app"></param>
+        /// <param name="env"></param>
```
Comment (ungb) [inline TestWebAppProject/Startup.cs]: # Does this work?

Comment (ungb) [reply TestWebAppProject/Startup.cs]:
```

Test

```

File: TestWebAppProject/Startup.cs [hunk 7]
```diff
@@ -31,6 +34,15 @@ public void ConfigureServices(IServiceCollection services)
...
+        /// <param name="loggerFactory"></param>
+
```
Comment (ungb) [inline TestWebAppProject/Startup.cs]: fooo

Comment (ungb) [inline TestWebAppProject/Startup.cs]: bar

File: TestWebAppProject/Startup.cs [hunk 8]
```diff
@@ -31,6 +34,15 @@ public void ConfigureServices(IServiceCollection services)
...
+
```
Comment (ungb) [inline TestWebAppProject/Startup.cs]: test

File: TestWebAppProject/Startup.cs [hunk 9]
```diff
@@ -31,6 +34,15 @@ public void ConfigureServices(IServiceCollection services)
             services.AddMvc();
         }
 
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="app"></param>
+        /// <param name="env"></param>
+        /// <param name="loggerFactory"></param>
```
Comment (ungb) [inline TestWebAppProject/Startup.cs]: Hello

Comment (ungb) [inline TestWebAppProject/Startup.cs]: World

File: TestWebAppProject/Startup.cs (hunk 7 above)
Comment (ungb) [inline TestWebAppProject/Startup.cs]: test

File: TestWebAppProject/Startup.cs [hunk 10]
```diff
@@ -31,6 +34,15 @@ public void ConfigureServices(IServiceCollection services)
             services.AddMvc();
         }
 
+
+        /// <summary>
+        /// 
+        /// </summary>
```
Comment (ungb) [inline TestWebAppProject/Startup.cs]: <!DOCTYPE HTML>
<html>
<head>
<title>Title of the document</title>
</head>

<body>
The content of the document......
</body>

</html>

File: TestWebAppProject/Startup.cs [hunk 11]
```diff
@@ -1,60 +1,75 @@
...
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
 
-        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
+namespace TestWebAppProject
+{
+    public class Program
+    {
+        public static void Main(string[] args)
```
Comment (meaghanlewis) [inline TestWebAppProject/Startup.cs:37]: Refer to this issue: https://github.com/github/VisualStudio/issues/1151

Comment (meaghanlewis) [inline TestWebAppProject/Startup.cs:37]: Refer to this comment: https://github.com/github/VisualStudio/issues/1151

File: TestWebAppProject/Startup.cs [hunk 12]
```diff
@@ -1,60 +1,75 @@
...
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
 
-        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
+namespace TestWebAppProject
```
Comment (meaghanlewis) [inline TestWebAppProject/Startup.cs:33]: Refer here: https://github.com/github/VisualStudio/issues/1151

File: TestWebAppProject/Startup.cs [hunk 13]
```diff
@@ -1,60 +1,75 @@
...
-        public IConfigurationRoot Configuration { get; }
+            var host = new WebHostBuilder()
+                .UseKestrel()
+                .UseContentRoot(Directory.GetCurrentDirectory())
+                .UseIISIntegration()
+                .UseStartup<Startup>()
+                .UseApplicationInsights()
+                .Build();
 
-        // This method gets called by tffhe runtime. Use this method to add services to the container.
```
Comment (meaghanlewis) [inline TestWebAppProject/Startup.cs:27]: comment

Comment (meaghanlewis) [inline TestWebAppProject/Startup.cs:27]: comment

File: TestWebAppProject/Startup.cs [hunk 14]
```diff
@@ -1,60 +1,75 @@
...
-            var builder = new ConfigurationBuilder()
-                .SetBasePath(env.ContentRootPath)
-                .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
-                .AddJsonFile($"appsettings.{env.EnvironmentName}.json", optional: true)
-                .AddEnvironmentVariables();
-            Configuration = builder.Build();
-        }
-
-        public IConfigurationRoot Configuration { get; }
+            var host = new WebHostBuilder()
```
Comment (meaghanlewis) [inline TestWebAppProject/Startup.cs:14]: comment 1

Comment (meaghanlewis) [inline TestWebAppProject/Startup.cs:14]: comment 2

Comment (meaghanlewis) [reply TestWebAppProject/Startup.cs:14]: reply

File: README.md [hunk 15]
```diff
@@ -0,0 +1,3 @@
+# TestWebAppProject
+
```
Comment (ungb) [inline README.md:2]: @tierninho Hey How's it going? 😄 
panda

Does this fix issue https://github.com/github/VisualStudio/issues/1163


[Check this out](https://github.com/marketplace)

File: TestWebAppProject/Startup.cs [hunk 16]
```diff
@@ -1,60 +1,75 @@
...
+                .UseKestrel()
```
Comment (meaghanlewis) [inline TestWebAppProject/Startup.cs:15]: blah

File: TestWebAppProject/Startup.cs [hunk 17]
```diff
@@ -1,60 +1,75 @@
...
+                .UseContentRoot(Directory.GetCurrentDirectory())
```
Comment (meaghanlewis) [inline TestWebAppProject/Startup.cs:16]: Comment

File: TestWebAppProject/Startup.cs [hunk 18]
```diff
@@ -1,60 +1,75 @@
 ﻿using System;
```
Comment (meaghanlewis) [inline TestWebAppProject/Startup.cs:1]: comment on line 1

File: README.md [hunk 19]
```diff
@@ -0,0 +1,3 @@
...
+This test project rocks
```
Comment (meaghanlewis) [inline README.md:3]: comment

File: TestWebAppProject/Startup.cs [hunk 20]
```diff
@@ -1,60 +1,75 @@
...
-            app.UseMvc(routes =>
-            {
-                routes.MapRoute(
-                    name: "default",
-                    template: "{controller=Home}/{action=Index}/{id?}");
-            });
+            host.Run();
         }
     }
 }
```
Comment (meaghanlewis) [inline TestWebAppProject/Startup.cs:75]: comment here

File: README.md [hunk 21]
```diff
@@ -0,0 +1,3 @@
+# TestWebAppProject
```
Comment (meaghanlewis) [inline README.md:1]: comment on line 1

**Pull request closed without merging**